### PR TITLE
chore(blog) replacing tag "crypto" with "market data"

### PR DESCRIPTION
**Pull request by amyshwang**
image

### CODE REVIEW

**Comment by amyshwang** [comment]
Question: I don't know if we need to add a redirect for `https://questdb.io/blog/tags/crypto/`. 
I feel it is not necessary because people click on the tab rather than saving it as a link?

**Comment by argshook** [comment]
> Question: I don't know if we need to add a redirect for `https://questdb.io/blog/tags/crypto/`. I feel it is not necessary because people click on the tab rather than saving it as a link?

@amyshwang after checking pageview metrics i think it's safe to say we don't need a redirect.

In March there were just 22 visits of `crypto` tag, compared to 58'011 total visits

File: blog/2021-02-18-realtime-crypto-tracker-with-kafka-and-questdb.mdx [hunk 1]
```diff
@@ -9,11 +9,11 @@ description:
   QuestDB, and calculate moving averages with Pandas.
 keywords:
   - timeseries
-  - crypto
+  - marketdata
   - kafka
   - pandas
   - python
-tags: [tutorial, python, pandas, kafka, crypto]
+tags: [tutorial, python, pandas, kafka, market data]
```
Comment (amyshwang) [inline blog/2021-02-18-realtime-crypto-tracker-with-kafka-and-questdb.mdx:16]:
```suggestion
tags: [tutorial, python, pandas, kafka, market data, crypto]
```

File: blog/2021-04-12-stream-ethereum-data.mdx [hunk 2]
```diff
@@ -13,9 +13,9 @@ keywords:
   - kafka
   - blockchain
   - ethereum
-  - crypto
+  - marketdata
   - etl
-tags: [tutorial, crypto, blockchain, ethereum, kafka, infura]
+tags: [tutorial, market data, blockchain, ethereum, kafka, infura]
```
Comment (amyshwang) [inline blog/2021-04-12-stream-ethereum-data.mdx:18]:
```suggestion
tags: [tutorial, market data, blockchain, ethereum, kafka, infura, crypto]
```

File: blog/2021-04-12-stream-ethereum-data.mdx [hunk 3]
```diff
@@ -13,9 +13,9 @@ keywords:
   - kafka
   - blockchain
   - ethereum
-  - crypto
+  - marketdata
```
Comment (amyshwang) [inline blog/2021-04-12-stream-ethereum-data.mdx:16]:
```suggestion
  - marketdata
  - crypto
```

File: blog/2021-02-18-realtime-crypto-tracker-with-kafka-and-questdb.mdx [hunk 4]
```diff
@@ -9,11 +9,11 @@ description:
   QuestDB, and calculate moving averages with Pandas.
 keywords:
   - timeseries
-  - crypto
+  - marketdata
```
Comment (amyshwang) [inline blog/2021-02-18-realtime-crypto-tracker-with-kafka-and-questdb.mdx:12]:
```suggestion
  - marketdata
  - crypto
```

File: blog/2022-02-10-questdb-google-data-studio.mdx [hunk 5]
```diff
@@ -10,13 +10,12 @@ description:
...
 keywords:
   - timeseries
-  - crypto
   - ethereum
   - solana
   - kaggle
   - googlecloud
   - marketdata
-tags: [tutorial, crypto, googlecloud, market data, autopilot]
+tags: [tutorial, market data, googlecloud, market data, autopilot]
```
Comment (amyshwang) [inline blog/2022-02-10-questdb-google-data-studio.mdx:18]:
```suggestion
tags: [tutorial, market data, crypto, googlecloud, market data, autopilot]
```

File: blog/2022-03-08-questdb-crypto-pandas.md [hunk 6]
```diff
@@ -10,12 +10,11 @@ description:
   records on the fly.
 keywords:
   - timeseries
-  - crypto
   - pandas
   - matplotlib
   - seaborn
   - marketdata
-tags: [tutorial, crypto, python, pandas, market data, matplotlib]
+tags: [tutorial, market data, python, pandas, market data, matplotlib]
```
Comment (amyshwang) [inline blog/2022-03-08-questdb-crypto-pandas.md:17]:
```suggestion
tags: [tutorial, market data, crypto, python, pandas, market data, matplotlib]
```

File: blog/2022-03-15-cryptocurrency-grafana-questdb.md [hunk 7]
```diff
@@ -11,9 +11,9 @@ keywords:
   - questdb
   - python
   - grafana
-  - crypto
+  - marketdata
```
Comment (amyshwang) [inline blog/2022-03-15-cryptocurrency-grafana-questdb.md:14]:
```suggestion
  - marketdata
  - crypto
```

File: blog/2022-03-15-cryptocurrency-grafana-questdb.md [hunk 8]
```diff
@@ -11,9 +11,9 @@ keywords:
...
   - database
-tags: [tutorial, crypto, trading, python, grafana]
+tags: [tutorial, market data, trading, python, grafana]
```
Comment (amyshwang) [inline blog/2022-03-15-cryptocurrency-grafana-questdb.md:16]:
```suggestion
tags: [tutorial, market data, crypto, trading, python, grafana]
```

File: blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx [hunk 9]
```diff
@@ -6,13 +6,13 @@ author_url: https://github.com/tris0laris
...
 keywords:
-  - crypto
+  - market data
   - bitcoin
   - timeseries
   - coinbase
   - grafana
   - sql
-tags: [demo, crypto, bitcoin, coinbase, grafana, opensource, candlestick]
+tags: [demo, market data, bitcoin, coinbase, grafana, opensource, candlestick]
```
Comment (amyshwang) [inline blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx:15]:
```suggestion
tags: [demo, market data, crypto, bitcoin, coinbase, grafana, opensource, candlestick]
```

File: blog/2022-10-03-athletex-interview.mdx [hunk 10]
```diff
@@ -8,8 +8,8 @@ description:
   This is a brand-new series for which we interviewed different developers in
   our community. For the post of this series, we interviewed Kevin Kamto,
   Co-founder at AthleteX Markets.
-keywords: [blockchain, defi, crypto, timeseries, database, mlb, athletex]
-tags: [community, story, devstory, crypto, market data]
+keywords: [blockchain, defi, market data, timeseries, database, mlb, athletex]
```
Comment (amyshwang) [inline blog/2022-10-03-athletex-interview.mdx:11]:
```suggestion
keywords: [blockchain, defi, market data, crypto, timeseries, database, mlb, athletex]
```

File: blog/2021-01-18-low-code-bitcoin-ticker-workflow-with-time-series-database.mdx [hunk 11]
```diff
@@ -12,8 +12,9 @@ keywords:
   - bitcoin
   - streaming
   - n8n
+  - marketdata
   - lowcode
-tags: [tutorial, low code, bitcoin, market data, streaming, n8n, crypto]
+tags: [tutorial, low code, bitcoin, market data, streaming, n8n]
```
Comment (amyshwang) [inline blog/2021-01-18-low-code-bitcoin-ticker-workflow-with-time-series-database.mdx:17]:
```suggestion
tags: [tutorial, low code, bitcoin, market data, streaming, n8n, crypto]
```

**Review by tris0laris — APPROVED** [review]

**Pull request merged**